tools/utility/dockerise/platform.py

```diff
@@ -116,11 +117,19 @@ def build(repository, platform):
         cprint("Docker build returned exit code {}".format(err), "red", attrs=["bold"])
         exit(err)
 
-    # If we were building the selected platform then update our selected tag
-    if _selected:
-        err = subprocess.run(
-            ["docker", "image", "tag", "{}:{}".format(repository, platform), "{}:selected".format(repository)],
-            stdout=subprocess.DEVNULL,
-        ).returncode
-        if err != 0:
-            raise RuntimeError("docker image tag returned a non-zero exit code")
+
+def pull(repository, platform):
+    # Define our tag strings
+    remote_tag = "{0}/{0}:{1}".format(repository, platform)
+    local_tag = "{0}:{1}".format(repository, platform)
+    selected_tag = "{}:selected".format(repository)
+
+    print("Pulling remote image", remote_tag)
+    err = subprocess.run(["docker", "pull", remote_tag]).returncode
+    if err != 0:
+        raise RuntimeError("docker image pull returned a non-zero exit code")
+
+    print("Tagging ", remote_tag, "as", local_tag)
```

Comment:
Well this seems like poor cohesion for a function named pull :P

Reply:
Pretty sure I had it named `pull_and_tag` and changed it for some reason I can't remember